gcloud/datastore/helpers.py

```diff
@@ -115,6 +115,37 @@ def _get_meaning(value_pb, is_list=False):
     return meaning
 
 
+def _new_value_pb(entity_pb, name):
+    """Add (by name) a new ``Value`` protobuf to an entity protobuf.
+
+    :type entity_pb: :class:`gcloud.datastore._generated.entity_pb2.Entity`
+    :param entity_pb: An entity protobuf to add a new property to.
+
+    :type name: string
+    :param name: The name of the new property.
+
+    :rtype: :class:`gcloud.datastore._generated.entity_pb2.Value`
+    :returns: The new ``Value`` protobuf that was added to the entity.
+    """
```

Comment:
Given that we always immediately set to value, maybe `_new_value_pb` should take the value argument and set it via `_set_protobuf_value`?